src/streaming/extensions/ProtectionExtensions.js

```diff
@@ -16,7 +16,17 @@ MediaPlayer.dependencies.ProtectionExtensions = function () {
     this.system = undefined;
     this.log = undefined;
     this.keySystems = [];
+
     this.clearkeyKeySystem = undefined;
+    // hard-coded set of keys for testing ClearKey
```

Comment:
I would prefer to have these provided by the page in the protectionData.
For the test content the matching key can be in the sources.json with the mpd url.